.github/workflows/code-quality.yaml

```diff
@@ -1,11 +1,32 @@
 ---
 name: Code static analysis
-on: [pull_request]  # yamllint disable-line rule:truthy
+"on": ["push", "pull_request"]
```

Comment:
This is segregated to standalone PR

* https://github.com/opendatahub-io/notebooks/pull/547